src/scripts/settings/webSettings.js

```diff
@@ -0,0 +1,17 @@
+define(['appStorage', 'events'], function (appStorage, events) {
+    'use strict';
+
+    var data = {};
+
+    function WebSettings() {
+        fetch("/config.json").then(function (response) {
+            data = response.json();
+        })
+    }
+
+    WebSettings.prototype.getMultiServer = function () {
```

Comment:
```suggestion
    WebSettings.prototype.allowMultiServer = function () {
```

Reply:
I figured we could use get here because we will never add a set method to these options.

Reply:
I still don't like the verb `get` here, because it looks a bit confusing to me what it "gets".
Maybe rename to `isMultiServerEnabled` to be extra-clear?